shared/src/persistence/dynamo/notifications/deleteUserConnection.js

```diff
@@ -17,6 +17,12 @@ exports.deleteUserConnection = async ({ applicationContext, connectionId }) => {
     applicationContext,
   });
 
+  if (!connection) {
```

Comment:
just double checking; these changes are ok to have right?  These came from stuff Mike merged to court/staging right?

Reply:
yep! the only changes associated with this bugfix are in `ExternalDocumentNonStandardB` and its test